lib/system/io.nim

```diff
@@ -186,6 +186,21 @@ proc readChars*(f: File, a: var openArray[char], start, len: Natural): int {.
   ## reads `len` bytes into the buffer `a` starting at `a[start]`. Returns
   ## the actual number of bytes that have been read which may be less than
   ## `len` (if not as many bytes are remaining), but not greater.
+  ##
+  ## runnableExamples("-r:off"):
```

Comment:
You need to remove the `##` for it to work :P
`runnableExamples` is a template, that is understood by the docgen tool.

Reply:
Oh woops, you're right thanks.